destination/parse.go

```diff
@@ -164,45 +164,46 @@ func (d deviceFinder) parseDeviceList() (*deviceList, error) {
 
 		for _, deviceList := range list.Devices {
 			for _, device := range deviceList {
-				if device.IsAvailable {
```

Comment:
Not sure what was the original idea behind this control flow, but I think it's better to loop through all devices and only print a warning for unavailable devices.

Reply:
The idea was to retry the simctl call if the device list is empty. This original simctl bug may be solved since. Checking the IsAvailable flag was just a case of trying to be too paranoid, I did not see any case in parctice where the device list is present but all of them are unavailable.

Reply:
I think we should remove the retry or move this check out of the retry loop, as with the changes it will not fail (trigger a retry) in the theoretical case when the device list is empty.